coil-base/src/main/java/coil/transform/RoundedCornersTransformation.kt

```diff
@@ -4,24 +4,26 @@ package coil.transform
 
 import android.graphics.Bitmap
 import android.graphics.BitmapShader
-import android.graphics.Color
 import android.graphics.Paint
-import android.graphics.PorterDuff
-import android.graphics.RectF
 import android.graphics.Shader
+import android.graphics.RectF
+import android.graphics.Color
+import android.graphics.PorterDuff
+import android.graphics.Path
 import androidx.core.graphics.applyCanvas
 import coil.bitmappool.BitmapPool
 
 /**
  * A [Transformation] that rounds the corners of an image.
  */
-class RoundedCornersTransformation(private val radius: Float) : Transformation {
+class RoundedCornersTransformation(private vararg val radii: Float) : Transformation {
```

Comment:
We should have two constructors instead of using `vararg`:

`RoundedCornersTransformation(radius: Float)` and `RoundedCornersTransformation(left: Float, top: Float, right: Float, bottom: Float)`

The first constructor can delegate to the second.